tests/CMakeLists.txt

```diff
@@ -154,6 +154,44 @@ add_test(NAME fiat_test_abor1
       "-DPASS_REGULAR_EXPRESSION=ABOR1.*aborting from OpenMP parallel region"
       -P ${CMAKE_CURRENT_SOURCE_DIR}/test_program_output.cmake )
 
+#-----------------------------------------------------------------------------------------
+# Test fiat_test_mpl_alltoallv
+#
+#
+
+ecbuild_add_executable(
+    TARGET  fiat-test-mpl-alltoallv-no-output
+    SOURCES test_mpl_alltoallv_no_output.F90 
+    LIBS    fiat
+    LINKER_LANGUAGE Fortran
+    NOINSTALL )
+
+set(LAUNCH2  ${MPIEXEC} ${MPI_OPTS} ${MPIEXEC_NUMPROC_FLAG} 2)
+add_test(NAME fiat_test_mpl_alltoallv_no_output
+         COMMAND ${CMAKE_COMMAND}
+         "-DEXECUTABLE=$<TARGET_FILE:fiat-test-mpl-alltoallv-no-output>"
+         "-DLAUNCH=${LAUNCH2}"
+         -P ${CMAKE_CURRENT_SOURCE_DIR}/test_mpl_alltoallv_no_output.cmake )
+
+#-----------------------------------------------------------------------------------------
+# Test test_linked_list
+#
+#
+
+ecbuild_add_executable(
+    TARGET  fiat-test-linked-list-no-output
+    SOURCES test_linked_list_no_output.F90
+    LIBS    fiat
+    LINKER_LANGUAGE Fortran
+    NOINSTALL )
+
+add_test(NAME fiat_test_linked_list_no_output
+         COMMAND ${CMAKE_COMMAND}
+         "-DEXECUTABLE=$<TARGET_FILE:fiat-test-linked-list-no-output>"
+         "-DLAUNCH=${LAUNCH}"
+         -P ${CMAKE_CURRENT_SOURCE_DIR}/test_linked_list_no_output.cmake )
+
+
```

Comment:
Same as for the other new test: no need for parsing output and remove "_no_output" suffix.